### ISSUE TITLE
brackets stopped responding when using bootstrap min css file

### ISSUE THREAD

**Issue by dinesh1d** [adobe/brackets#10950]
using brackets as text editor. Bootstrap.min css file is stopped
responding on brackets. Html file is working fine, but when switch on to css file brackets hangs. Brackets worked fine on restart. When css file loads the problem started again. The file works fine on other text editors, but not on brackets. Apart from bootstrap css other css files works on brackets. disabled the extensions in brackets and tried again. it's still not working.

Bootstrap version 3.3.2
Brackets version 1.1
Windows7

https://drive.google.com/open?id=0B7jY3Ri5R7prQ3BWRzNuOWpzN1k&authuser=0

**Comment by peterflynn** [comment]
@dinesh1d The Google Drive link doesn't have permissions to view/download the CSS file.  Can you upload it as a public [Gist](https://gist.github.com/) maybe?

Are you editing the bootstrap.min.css file, or just viewing it?  Are you putting your cursor in a specific place within the file, or running any other specific Brackets command (for example, Quick Edit) before the freeze occurs?

**Comment by MarcelGerber** [comment]
@peterflynn You can get it here: https://raw.githubusercontent.com/twbs/bootstrap/v3.3.2/dist/css/bootstrap.min.css

It's quite possible @dinesh1d sees #10879 (I see that too with that Bootstrap file and with extensions disabled), which would be fixed with my PR #10882.

**Comment by dinesh1d** [comment]
@peterflynn I would upload it on gist. Yes i am editing bootstrap for customizing the page. sometime when using ctrl+f command on brackets, it's freezes after showing first few results. able to scroll page for a few minutes then it's hangs.

**Comment by dinesh1d** [comment]
bootstrap.css, bootsrap.theme.css, bootstrap.theme.min.css all these file worked fine on brackets. only bootstrap.min.css is causing problems.

**Comment by CHALLENGER22** [comment]
yes i too face this same problem with bootstrap min css. when i open BS min css to view/edit Brackets showing the file but after some scroll it hangs & then i have to end the process to close Bracket.

My pc configuration
Window 7 64bit
Intel 950 i7 processor
6gb ram
MSI 460gtx x 2 in SLI

Bracket is a very good editor but please try to solve this problem & make Bracket handle vary large coding files smoothly.. Thank in Advance :)

**Comment by ThinkWired** [comment]
+1

Same issue here.
